docs/docsite/rst/dev_guide/developing_locally.rst

```diff
@@ -43,7 +43,7 @@ Once you save your module file in one of these locations, Ansible will load it a
 
 To confirm that ``my_custom_module`` is available:
 
-* type ``ansible-doc -t module my_custom_module``. You should see the documentation for that module.
+* type ``ansible localhost -m my_custom_module``. You should see the output for that module.
```

Comment:
Could you please add 

```suggestion
* type ``ansible localhost -m my_custom_module``. You should see the output for that module.

or 

* type ``ansible-doc -t module my_custom_module``. You should see the documentation for that module.
```

Since the next note section talks about `ansible-doc` and module documentation as well. Removing `ansible-doc` will make that note irrelevant.